tests/test_group.rs

```diff
@@ -151,27 +139,18 @@ fn basic_group_setup() {
         Credential::Basic(alice_credential),
         None,
     );
-    let alice_key_package = alice_key_package_bundle.get_key_package();
-
-    let charlie_key_package_bundle = KeyPackageBundle::new(
-        &ciphersuite,
-        &alice_identity.get_signature_key_pair().get_private_key(), // TODO: bad API, we shouldn't have to get the private key out here (this function shouldn't exist!)
-        Credential::Basic(charlie_credential),
-        None,
-    );
-    let charlie_key_package = charlie_key_package_bundle.get_key_package();
 
     // Alice creates a group
     let group_id = [1, 2, 3, 4];
-    let mut group_alice_1234 = MlsGroup::new(&group_id, ciphersuite, alice_key_package_bundle);
+    let group_alice_1234 = MlsGroup::new(&group_id, ciphersuite, alice_key_package_bundle);
 
     // Alice adds Bob
     let bob_add_proposal = group_alice_1234.create_add_proposal(
         group_aad,
         &alice_identity.get_signature_key_pair().get_private_key(),
         bob_key_package.clone(),
     );
-    let commit = match group_alice_1234.create_commit(
+    let _commit = match group_alice_1234.create_commit(
```

Comment:
Yeah this test needs to get finished to get rid of the compiler warnings. But this way we get less yellow output until we're there 😄